storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -43,15 +51,22 @@ Always also set `aria-required` on the individual radio buttons though, to make
 
 <Canvas of={FieldSetStories.RadioGroup} />
 
-### Radio group with error
+### Radio Group With Not Required Label
+
+A Field Set with a radio button group can also indicate whether all of the radio inputs it contains are <em>not</em> required.
```

Comment:
We decided to use ‘Radio’ everywhere in another PR, not ‘radio input’ or ‘radio button’.